README.md

```diff
@@ -36,19 +36,51 @@ module.exports = {
 }
 ```
 
-Here's the full list of [`remark` plugins][1]. Note that [`remark-html`][3] 
-is always included as the last plugin and should not be included in the 
+Here's the full list of [`remark` plugins][1]. Note that [`remark-html`][3]
+is always included as the last plugin and should not be included in the
 `plugins` list.
 
-> This loader makes use of the [`html-loader`][5] under the hood.
+Frontmatter values are assigned to an object exported as `attributes`. For
+example, if you have this markdown file:
+
+```md
+---
+title: a clever remark
+---
+hello world!
+```
+
+You can use it like this:
+
+```js
+import contents, { attributes } from './test.md'
+
+console.log(contents) // logs "hello world!"
+console.log(attributes) // logs "{ title: 'a clever remark' }"
+```
```

Comment:
I was going by what's already in the library, which already parses frontmatter via the `front-matter` package. Agreed that it could be better to leave that to userspace as a remark plugin! (That said, I think the use of frontmatter in markdown files is so common that it could be useful to include it as an opinionated default, similar to how `remark-html` is included now.)

Reply:
🤦‍♂️ wow that shows how long it's been since I touched this. Completely forgot that, I must've added that as a quick hack before I knew about the plug-in. Would definitely be good to get rid of that and somehow this via the plug-in instead. Just to clarify, there already is a frontmatter plugin for remark, I linked it above.